flow/view_holder.cc

```diff
@@ -150,22 +127,47 @@ void ViewHolder::set_hit_testable(bool value) {
 }
 
 void ViewHolder::set_focusable(bool value) {
-  view_properties_.focus_change = value;
+  if (view_properties_.focus_change != value) {
+    view_properties_.focus_change = value;
+    view_properties_changed_ = true;
+  }
 }
 
 void ViewHolder::set_size(const SkSize& size) {
+  if (size.width() > 0.f && size.height() > 0.f) {
+    if (view_properties_.bounding_box.max.x != size.width()) {
```

Comment:
Should we be concerned with small rounding differences here since these are working on floats?  If I change my width from 100.0 to 100.001 this won't update. I'm not sure if we really want to update the view properties if it is that small of a change but it is something we should be aware of.

Reply:
This should work fine with a change that small, it would start to break down with change smaller that 1e-7 but other places in the flutter engine would have that problem too